client/me/purchases/remove-purchase/index.jsx

```diff
@@ -261,6 +267,10 @@ class RemovePurchase extends Component {
 			return this.renderDomainDialog();
 		}
 
+		if ( isDomainMapping( purchase ) ) {
+			return this.this.renderPlanDialog();
```

Comment:
THIS THIS!!!!! :scream:

Reply:
Updated it. :laughing:

Reply:
Would you believe me if I said I did it on purpose because double referencing like this is a workaround for a bug in IE 6 on Windows Me?

No? Fine, have it your way, but don't come crying to me when our IE 6 Win Me users complain!

Reply:
![Image URL](https://user-images.githubusercontent.com/1103398/73274048-36260900-41e5-11ea-84c3-c5f06299d8d7.jpg)

Reply:
![386a007910aa3434d7b7e4553b36991961d99f452b915a2ad38f8c894f9b8207](https://user-images.githubusercontent.com/1182160/73274270-99b03680-41e5-11ea-9359-117eff38b766.jpg)